File: workshop/oddecho.py
numberOfWords = int(input())

# num_linjer = input()
# l1 = input()
# l2 = input()
# l3 = input()
# l4 = input()
# l5 = input()
# print(l1)
# print(l3)
# print(l5)

odd = True

while (numberOfWords > 0):
    word = input()
    if (odd):
        print(word)

    odd = not odd
    numberOfWords = numberOfWords - 1

File: workshop/oddecho.java
package workshop;

import java.util.Scanner;

/**
 * Write a description of class OddEcho here.
 *
 * @author (your name)
 * @version (a version number or a date)
 */
public class oddecho
{
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        
        int numOfWords = s.nextInt();
        /*
        String s1 = s.next();
        String s2 = s.next();
        String s3 = s.next();
        String s4 = s.next();
        String s5 = s.next();
        
        System.out.println(s1);
        System.out.println(s3);
        System.out.println(s5);
         */
        boolean odd = true;

        while (numOfWords > 0) {
            String word = s.next();

            if (odd) {
                System.out.println(word);
            }

            odd = !odd;
            numOfWords = numOfWords - 1;
        }
    }
}
